articles/azure/azs-how-create-S2S-VPN.md

```diff
@@ -0,0 +1,223 @@
+
+   - **Configure BGP settings** - Use only when configuring BGP.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+   ![Create new local network gateway](images/azs-browser-create-Lnetwork-gateway.png)
+
+5. Click **Create**.
+
+## Create the VPN connection
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your on-premises VPN device:
+
+1. Navigate to your virtual network gateway by clicking on **All services** in the favourites panel, then selecting **Virtual network gateways** under the networking section.
+
+2. Select your virtual network gateway from the list.
+
+3. In the blade for your virtual network gateway, select **Connections** under the **Settings** section.
+
+4. Click the **+Add** button.
+
+5. In the **Add connection** blade, enter the following information:
+
+   - **Name** - The name of the connection.
+
+   - **Connection type** - Select **Site-to-site (IPSec)**.
+
+   - **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   - **Local network gateway**- Select the local network gateway you created earlier.
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways.. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - When adding a connection for an existing peer, the resource group can't be changed.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+   ![Add new connection](images/azs-browser-add-connection.png)
+
+6. Click **OK**.
+
+## Verify the VPN connection
+
+After configuring the VPN device on your local network, you can verify the VPN connection with the following steps:
+
+1. Navigate to your virtual network gateway by clicking **All services** in the favourites panel, then selecting **Virtual network gateways** under the networking section.
+
+2. Select your virtual network gateway from the list.
+
+3. In the blade for your virtual network gateway, select **Connections** under the **Settings** section.
+
+4. Select your connection from the list.
+
+5. In the blade that has opened, you can view more information about your connection. The connection status will be **Succeeded** or **Connected** if there is a successful connection.
+
+## Feedback
+
+Each article should end with a feedback section that includes the following text:

```

Comment:
Remove this paragraph (it's just an instruction within the article template)

Reply:
Oops, missed that. Fixed